components/eamxx/src/physics/mam/mam_coupling.hpp

```diff
@@ -574,7 +576,8 @@ mam4::Prognostics aerosols_for_column(const AerosolState& dry_aero,
     progs.n_mode_c[m] = ekat::subview(dry_aero.cld_aero_nmr[m], column_index);
     for (int a = 0; a < num_aero_species(); ++a) {
       if (dry_aero.cld_aero_mmr[m][a].data()) {
-        progs.q_aero_c[m][a] = ekat::subview(dry_aero.cld_aero_mmr[m][a], column_index);
+        int prg_idx = mam4::utils::get_prognostics_index(m,a); //Index of HAERO's prognostics array
```

Comment:
we do not need `mam4::utils::get_prognostics_index(m,a)`.

Reply:
I have removed it.